templates/zerver/help/see-if-user-here-away.md

```diff
@@ -0,0 +1,35 @@
+### User List
+
+You can see if a user is active on Zulip by looking at the user list located on the far right side of your screen.
```

Comment:
The lines in your document are long and it's hard to read them - could you make sure your lines are 76 characters or less? It makes it easier to read them in the terminal.